en/documentation/index.md

```diff
@@ -52,7 +56,7 @@ the [installation guide](installation/) for help on installing Ruby.
 ### Reference Documentation
 
 [Official API Documentation][docs-rlo-en]
-: The official Ruby API documentation for different versions including
+: The official Ruby language documentation for different versions including
   the currently unreleased (trunk) version.
 
 [Ruby Core Reference][13]
```

Comment:
Both link 13 and 15 are pointing to outdated source on `ruby-doc.org`: `https://ruby-doc.org/3.2.2/`, which has the same content as `docs.ruby-lang.org`. Since they don't serve additional information nor have better UX (IMO), I think we can remove them to shorten the list.

Additionally, link 17 (https://rubydocs.org/) is blank and link 18 (https://apidock.com/) only has documentation up to 2.6. So they should be removed as well.

(I'm happy to open a separate PR to remove them if it's out of the scope of this PR)